GetVisibleServices/handler.ts

```diff
@@ -37,7 +39,8 @@ type IGetVisibleServicesHandler = () => Promise<IGetVisibleServicesHandlerRet>;
  */
 export function GetVisibleServicesHandler(
   blobService: BlobService,
-  onlyNationalService: boolean
+  onlyNationalService: boolean,
+  localServicesLimit?: NonNegativeInteger
```

Comment:
I'm wondering about these 2 configurations. Can we collapse this variables into one? As i intended the need, we can keep only `localServicesLimit`, changing the semantic of the variable indeed.Let's say If `localServicesLimit === undefined` keep all records, otherwise keep only limited local services. What do you think about?

Reply:
Fixed in 45f9a51. Now is needed to set `FF_LOCAL_SERVICES_LIMIT` at `0` to keep the default behaviour unchanged.